### PR TITLE
Implemented tags for blog posts

### PR DESCRIPTION
**Pull request by Majkl578**
Here's an idea of tags for articles, as suggested in #214. What do you think?

in blog tab:
list
in article detail:
article

Yes, visual styling is just a proof of concept, feel free to tweak it more.

TODO:

- [ ] L10N

### CODE REVIEW

File: source/_data/tagSet.neon [hunk 1]
```diff
@@ -0,0 +1,11 @@
+tagSet:
```
Comment (Majkl578) [inline source/_data/tagSet.neon]: This name sucks, but "tags" collides with local variable in templates. :/ Advice?

File: source/assets/css/style.css [hunk 2]
```diff
@@ -240,6 +240,14 @@ h1, h2, .h2, h3 {
     line-height: 2.5em;
 }
 
+.tags .tag {
+    margin: 0 3px;
+    padding: 4px 7px;
+    border-radius: 7px;
+    background: #ddd;
+	font-size: 13px;
```
Comment (enumag) [inline source/assets/css/style.css]: indentation

**Comment by TomasVotruba** [comment]
I need to see filtering as well. When do you plan to ad it?
Same might be done for authors as well.

**Comment by Majkl578** [comment]
> I need to see filtering as well. When do you plan to ad it?

Well, currently I have no idea how to do that easily with the tool used for generating the static output...

**Comment by TomasVotruba** [comment]
Yea, it requires mindset shift from dynamic PHP to one-way generated code. 

Me neither :D 

But, you can inspire here at Sculpin skeleton blog: https://github.com/sculpin/sculpin-blog-skeleton/tree/master/source/blog

**Comment by Majkl578** [comment]
>  It requires bmindset shift from dynamic PHP to one-way generated code.

My first idea was to create tags directory with separate latte files for each tag, but that sounds really dumb. :)

**Comment by TomasVotruba** [comment]
@Majkl578 Personally, I just don't like tags, categories... never used them in real life.

**Comment by TomasVotruba** [comment]
Could you rather create "related articles" section bellow each post? That's the place where action step would be great.

**Comment by TomasVotruba** [comment]
We can discuss this in person. Closing

**Pull request closed without merging**